ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -226,6 +283,12 @@ public static void classTearDown() {
   public void setUp() {
     vertx = Vertx.vertx();
     client = createClient();
+    makeKsqlRequest("CREATE SOURCE CONNECTOR " + TEST_CONNECTOR + " WITH ('connector.class'='" + MOCK_SOURCE_CLASS + "');");
```

Comment:
I don't think this belongs here. Most tests don't need the connector so it feels wasteful. How about we wrap this in a `givenConnectorExists()` method and only call it from the `Given:` portion of the tests that actually need it instead?